
Extracted contents to /scratch/hy2611/ML_Competition/dataset
Extracted contents to /scratch/hy2611/ML_Competition/dataset
/ext3/miniconda3/lib/python3.10/site-packages/torch/optim/lr_scheduler.py:139: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
Traceback (most recent call last):
  File "/scratch/hy2611/ML_Competition/main.py", line 109, in <module>
    main()
  File "/scratch/hy2611/ML_Competition/main.py", line 91, in main
    wandb.log({"loss": loss.item()}, {'lr': scheduler.get_last_lr()}, step=epoch+1)
  File "/ext3/miniconda3/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 420, in wrapper
    return func(self, *args, **kwargs)
  File "/ext3/miniconda3/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 371, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/ext3/miniconda3/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 361, in wrapper
    return func(self, *args, **kwargs)
TypeError: Run.log() got multiple values for argument 'step'